src/lib/ports.ts

```diff
@@ -21,20 +21,38 @@ export type Port = {
   categories: string[];
   name: string;
   platform: Platform[];
-
-  color?: ColorName;
+  color: ColorName;
   icon?: string;
   alias?: string;
   url?: string;
   links?: Link[];
   upstreamed?: boolean;
 };
 
-export const yml = (await fetch("https://github.com/catppuccin/catppuccin/raw/main/resources/ports.yml")
+export type Userstyle = {
+  name: string;
+  categories: string[];
+  color: ColorName;
+  icon?: string;
+  readme: Record<string, unknown>;
```

Comment:
This isn't the full type but it is enough for our purposes. We have the actual type definitions in each respective repository, how should we pull those in?